cmd/otel-agent/main.go

```diff
@@ -72,6 +73,8 @@ func main() {
 		),
 		fx.Provide(newForwarderParams),
 		demultiplexer.Module,
+		orchestratorForwarderImpl.Module,
+		fx.Supply(orchestratorForwarderImpl.NewDefaultParams()),
```

Comment:
I believe it should be disabled in the otel agent (like the security agent)

Reply:
It was previously enabled by calling [aggregator.DefaultAgentDemultiplexerOptions()](https://github.com/DataDog/datadog-agent/pull/21058/files/f7014d3f57f58aa03b9c818e66990af0263a9918#diff-096d5f40e9620faf4ab3841993a02469bc4ab3a8f202a2b74f9165d1c8fe16baR80) which set [UseOrchestratorForwarder: true](https://github.com/DataDog/datadog-agent/pull/21058/files/f7014d3f57f58aa03b9c818e66990af0263a9918#diff-c363b00939bb2af8b9f63471bd173091aa6335b59371d59a2c83514c3f5e698cL91)

Reply:
I have got the confirmation from open telemetry team and it should not be enabled.